readme.md

```diff
@@ -116,17 +117,17 @@ When decoding the message, each pair of numbers is translated using the coordina
 
 The `polybius()` function in the `src/polybius.js` file has two parameters:
 
-- _input_ refers to the inputted text to be encoded or decoded.
-- _encode_ refers to whether you should encode or decode the message. By default it is set to `true`.
+- _input_ is a string that refers to the inputted text to be encoded or decoded.
+- _encode_ is a boolean that refers to whether you should encode or decode the message. By default it is set to `true`.
 
 When building the function, keep the following constraints and rules in mind:
 
 - You are welcome to assume that _no additional symbols will be included as part of the input._ Only spaces and letters will be included.
 - When encoding, your output should _still be a string._
 - When decoding, the number of characters in the string _excluding spaces_ should be even. Otherwise, return `false`.
-- Spaces should be maintained throughout.
-- Capital letters can be ignored.
-- The letters "I" and "J" share a space. When encoding, both letters can be converted to `42`, but when decoding, both letters should somehow be shown.
+- Spaces in the message should be maintained before and after encoding or decoding.
+- Encoding is case-insensitive (e.g., both "a" or "A" would be encoded to the same result).
```

Comment:
Would love to know if I interpreted "Capital letters can be ignored' correctly with this change.